sopel/web.py

```diff
@@ -86,7 +86,7 @@ def get(uri, timeout=20, headers=None, return_headers=False,
         headers = default_headers
     else:
         tmp = default_headers.copy()
-        headers = tmp.update(headers)
+        tmp.update(headers)
```

Comment:
So tmp.update(headers) returns nothing, so previously headers was being set to nothing.
Just realized that I probably should have added a headers = tmp then, but I'm not sure why there is a copying of the headers to a tmp variable in the first place...
This does occur again in another spot in web.py which I neglected to update.